modules/delivery/delivery.service.js

```diff
@@ -0,0 +1,244 @@
+const request = require('request');
```

Comment:
We have `axios`, use it for making an HTTP request, it is much better than default `request`

Reply:
fixed